Public/Src/Engine/Processes/SandboxedProcessReports.cs

```diff
@@ -690,11 +767,14 @@ private bool FileAccessReportLineReceived<T>(ref T data, FileAccessReportProvide
                 return true;
             }
 
-            FileAccessStatusMethod method = FileAccessStatusMethod.PolicyBased;
+            // If this is an augmented file access, the method was not based on policy, but a trusted tool reported the access
+            FileAccessStatusMethod method = isAnAugmentedFileAccess? FileAccessStatusMethod.TrustedTool : FileAccessStatusMethod.PolicyBased;

```

Comment:
```suggestion
            FileAccessStatusMethod method = isAnAugmentedFileAccess ? FileAccessStatusMethod.TrustedTool : FileAccessStatusMethod.PolicyBased;
```

Reply:
nit: I don't like Lance's ternary-code style.
Either this
```
FileAccessStatusMethod method = isAnAugmentedFileAccess ?  FileAccessStatusMethod.TrustedTool  : FileAccessStatusMethod.PolicyBased;
```
or
```
FileAccessStatusMethod method = isAnAugmentedFileAccess 
    ?  FileAccessStatusMethod.TrustedTool  
    : FileAccessStatusMethod.PolicyBased;
```

---
In reply to: [343779106](https://github.com/microsoft/BuildXL/pull/1175#discussion_r343779106) [](ancestors = 343779106)